bin/abyss-pe

```diff
@@ -644,7 +644,7 @@ endif
 sealer_ks?=-k90 -k80 -k70 -k60 -k50 -k40 -k30
 
 %-8_scaffold.fa: %-8.fa
-	abyss-sealer -v -j$j --print-flanks -o$*-8 -S$< $(sealer_ks) $(SEALER_OPTIONS) $(in) $(se)
+	abyss-sealer -v -j$j --print-flanks -o $*-8 -S $< $(sealer_ks) $(SEALER_OPTIONS) $(in) $(se)
```

Comment:
I think the issue with the whitespace changes was tat the options did not parse correctly on my machine without them... It's been a month, so my memory is not crystal clear about this point however.